app/models/pipeline_run.rb

```diff
@@ -194,6 +194,11 @@ class PipelineRun < ApplicationRecord
 
   delegate :status_url, to: :sample
 
+  enum pipeline_type: {
```

Comment:
As I read this, I think I agree with @davidrissato that pipeline type might be a bit to generic (e.g. pipeline type seems to mean more like phylo tree vs. alignment pipeline ).
I think he proposed `pipeline_execution_strategy`.
Wdyt?

Reply:
I wasn't aware that there was a phylo tree pipeline, and I figured "alignment" is a stage. But if it makes it easier for us to understand, sure, since it's only going to be seen by devs anyways

Reply:
Naming of variables is always fo devs :)
i used `alignment` as general way to refer to the regular pipeline.